taxonomies/ingredients.txt

```diff
@@ -9997,7 +10149,7 @@ az:Qatılaşdırılmış süd
 be:згушчанае малако
 ca:llet condensada
 cs:kondenzované mléko
-de:Kondensmilch, Dosenmilch, Büchsenmilch
+de:Kondensmilch, Dosenmilch, Büchsenmilch, eingedickte Milch
```

Comment:
Here it is still …

Reply:
I remember now. There is a difference between evaporated and condensed milk (need to add some comments).